mm/2s2h/Rando/DrawFuncs.cpp

```diff
@@ -22,7 +24,13 @@ s32 EnMinifrog_OverrideLimbDraw(PlayState* play, s32 limbIndex, Gfx** dList, Vec
 
 // Other Actor Includes
 /* Minifrog */  #include "objects/object_fr/object_fr.h"
-// clang-format on
+/* Clock */     #include "overlays/actors/ovl_Obj_Tokeidai/z_obj_tokeidai.h"
+    // clang-format on
```

Comment:
I think the spaces here caused clang to miss the tag and not format the rest of the file.
```suggestion
// clang-format on
```

Reply:
Weird. I did that and the script still formatted it strangely. Not sure whats up with that, but we could move  // clang-format on to be beneath ObjTokeidai_RotateOnHourChange if you agree.